src/pybind/mgr/dashboard/frontend/src/styles/bootstrap-extends.scss

```diff
@@ -61,6 +65,12 @@ cd-health {
     &.cd-capacity-card {
       @extend .col-xl;
     }
+
+    &.cd-chart-card {
+      @extend .col-md-12;
+      @extend .col-xl-4;
+      @extend .col-2xl-3;
+    }
```

Comment:
I would suggest adding the following to reduce the wasted space while in `lg` breakpoint:
```suggestion
    &.cd-capacity-card {
      @extend .col-lg-3;
    }

    &.cd-performance-card {
      @extend .col-lg-6;
    }
    
    &.cd-chart-card {
      @extend .col-md-12;
      @extend .col-lg-6;
      @extend .col-xl-4;
      @extend .col-2xl-3;
    }
```

Before:
![image](https://user-images.githubusercontent.com/399326/89528870-4e519300-d7db-11ea-8273-fb503cd90168.png)
After:
![image](https://user-images.githubusercontent.com/399326/89528887-54477400-d7db-11ea-9033-9237f2b4f6de.png)

Reply:
Done. It definitely looks better for that breakpoint.